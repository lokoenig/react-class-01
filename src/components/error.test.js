import React from "react";
import { render, screen } from '@testing-library/react';
import { BrowserRouter } from "react-router-dom";

import ErrorPage from "./error";


describe('404 page', () => {
    it('renders ErrorPage component', () => {
        const { getByText, asFragment } = render(
            <BrowserRouter>
                <ErrorPage />
            </BrowserRouter>);
    //    screen.debug();
        expect(asFragment()).toMatchSnapshot();
    });
});