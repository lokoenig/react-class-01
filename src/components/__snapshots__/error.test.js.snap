// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page renders ErrorPage component 1`] = `
<DocumentFragment>
  <div
    id="error-page"
  >
    <h1>
      Oops!
    </h1>
    <p>
      Welcome to the 404.
    </p>
    <a
      href="/"
    >
      Back to home
    </a>
  </div>
</DocumentFragment>
`;
